index.html

```diff
@@ -0,0 +1,149 @@
+            }else{
+                if(document.getElementById("calc").value.includes('.')){
+                    //stops user from entering multiple decimals
+                    document.getElementById("calc").value=document.getElementById("calc").value;
+                } else{
+                    //adds decimal to end of value
+                    document.getElementById("calc").value = document.getElementById("calc").value + ".";
+                }
+            }
+        }
+
+        function myOperator(myOp){
+            if(op !== undefined && val1 !== undefined){
+                //skip to calculate if val1 and op are defined
+                calculate();
+            } else{
+                if(op==undefined){
+                    //sets val1 to what is currently inputted. sets operator to user selected option
+                    val1 = document.getElementById("calc").value;
+                    op=myOp;
+                    document.getElementById("calc").value = myOp;
+                } else {
+                    //overides previous operation in case user deviates from original selected operator
+                    op=myOp;
+                    document.getElementById("calc").value = myOp;
+                }
+            }
+        }
+
+        function calculate(){
+            if(val1 == undefined || op == undefined || isNaN(document.getElementById("calc").value)){
+                //do nothing if all variables are not defined or input value is not a number.
+                    //!!!Val2 is equal is in the input field at time of calculation
+            } else {
+                document.getElementById("calc").value = "...Loading answer";
+                val1 =undefined;
+                val2 =undefined;
+                op = undefined;
+            }
+        }
+        </script>
+
+
+        <div class="calculator buttons">
+
+            <button class="button command" id="clear" onclick="clearNum()">C</button>
+            <button class="button command" id="allClear" onclick="allClearNum()">AC</button>
+            <button class="button command" id="negative" onclick="makeNeg()">-/+</button>
+            <button class="button operator" id="divide"  onclick="myOperator('&#247;')">&#247;</button>
+
+            <button class="button number" id="numb" onclick="inputNum(7)">7</button>
+            <button class="button number" id="numb" onclick="inputNum(8)">8</button>
+            <button class="button number" id="numb" onclick="inputNum(9)">9</button>
+            <button class="button operator" id="multiply" onclick="myOperator('&#215;')">&#215;</button>
+
+            <button class="button number" id="numb" onclick="inputNum(4)">4</button>
+            <button class="button number" id="numb" onclick="inputNum(5)">5</button>
+            <button class="button number" id="numb" onclick="inputNum(6)">6</button>
+            <button class="button operator" id="subtract" onclick="myOperator('-')">-</but
+                ton>
```

Comment:
This is a bit odd looking, can we clean this up by having the whole closing tag on the same line?

Reply:
Typo. Fixed now.